MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -576,26 +590,41 @@ private void OnCreateFromPrefab(CreateFromPrefab payload, Action onCompleteCallb
         private void ProcessCreatedActors(CreateActor originalMessage, IList<Actor> createdActors, Action onCompleteCallback)
         {
             var guids = new DeterministicGuids(originalMessage.Actor?.Id);
+            var rootActor = createdActors.FirstOrDefault();
+
+            ProcessActors(rootActor.transform, rootActor.transform.parent.GetComponent<Actor>());
+
+            rootActor?.ApplyPatch(originalMessage.Actor);
```

Comment:
>rootActor? [](start = 12, length = 10)

Actor is a MonoBehaviour. Does that make this use of the `?` operator invalid here? (based on your comment in MixedRealityExtensionApp.cs)